src/www/ui_components/app-root.html

```diff
@@ -461,50 +466,50 @@
-            sr: {id: "sr", dir: "ltr"},
-            "sr-Latn": {id: "sr-Latn", dir: "ltr"},
-            sv: {id: "sv", dir: "ltr"},
-            th: {id: "th", dir: "ltr"},
-            tr: {id: "tr", dir: "ltr"},
-            uk: {id: "uk", dir: "ltr"},
-            ur: {id: "ur", dir: "rtl"},
-            vi: {id: "vi", dir: "ltr"},
-            zh: {id: "zh", dir: "ltr"},
-            "zh-CN": {id: "zh-CN", dir: "ltr"},
-            "zh-TW": {id: "zh-TW", dir: "ltr"},
+            am: {id: "am", name: "አማርኛ", dir: "ltr"},
+            ar: {id: "ar", name: "العربية", dir: "rtl"},
+            bg: {id: "bg", name: "Български", dir: "ltr"},
+            ca: {id: "ca", name: "Català", dir: "ltr"},
+            cs: {id: "cs", name: "Česky", dir: "ltr"},
+            da: {id: "da", name: "Dansk", dir: "ltr"},
+            de: {id: "de", name: "Deutsch", dir: "ltr"},
+            el: {id: "el", name: "Ελληνικά", dir: "ltr"},
+            en: {id: "en", name: "English", dir: "ltr"},
+            "es-419": {id: "es-419", name: "Español", dir: "ltr"},
+            fa: {id: "fa", name: "فارسی", dir: "rtl"},
+            fi: {id: "fi", name: "Suomi", dir: "ltr"},
+            fil: {id: "fil", name: "Wikang Filipino", dir: "ltr"},
+            fr: {id: "fr", name: "Français", dir: "ltr"},
+            he: {id: "he", name: "עברית", dir: "rtl"},
+            hi: {id: "hi", name: "हिन्दी", dir: "ltr"},
+            hr: {id: "hr", name: "Hrvatski", dir: "ltr"},
+            hu: {id: "hu", name: "Magyar", dir: "ltr"},
+            id: {id: "id", name: "Bahasa Indonesia", dir: "ltr"},
+            it: {id: "it", name: "Italiano", dir: "ltr"},
+            ja: {id: "ja", name: "日本語", dir: "ltr"},
+            km: {id: "km", name: "ភាសាខ្មែរ", dir: "ltr"},
+            ko: {id: "ko", name: "한국어", dir: "ltr"},
+            lt: {id: "lt", name: "Lietuvių", dir: "ltr"},
+            lv: {id: "lv", name: "Latviešu", dir: "ltr"},
+            nl: {id: "nl", name: "Nederlands", dir: "ltr"},
+            no: {id: "no", name: "Norsk (bokmål / riksmål)", dir: "ltr"},
+            pl: {id: "pl", name: "Polski", dir: "ltr"},
+            "pt-BR": {id: "pt-BR", name: "Português", dir: "ltr"},
+            ro: {id: "ro", name: "Română", dir: "ltr"},
+            ru: {id: "ru", name: "Русский", dir: "ltr"},
+            sk: {id: "sk", name: "Slovenčina", dir: "ltr"},
+            sl: {id: "sl", name: "Slovenščina", dir: "ltr"},
+            sr: {id: "sr", name: "Српски", dir: "ltr"},
+            "sr-Latn": {id: "sr-Latn", name: "Srpski", dir: "ltr"},
+            sv: {id: "sv", name: "Svenska", dir: "ltr"},
+            th: {id: "th", name: "ไทย", dir: "ltr"},
+            tr: {id: "tr", name: "Türkçe", dir: "ltr"},
+            uk: {id: "uk", name: "Українська", dir: "ltr"},
+            ur: {id: "ur", name: "اردو", dir: "rtl"},
+            vi: {id: "vi", name: "Việtnam", dir: "ltr"},
+            zh: {id: "zh", name: "简体中文", dir: "ltr"},
```

Comment:
Remove like suggested in the Manager.
@alalamav to weigh in.

Reply:
[+1](https://github.com/Jigsaw-Code/outline-server/pull/696#discussion_r442324076)

Reply:
Done